From 23763e6c58b940f5c32310461bade62c6fa22788 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Tue, 25 Dec 2012 10:52:06 +0800
Subject: [PATCH 29/54] ENGR00238201-2 V4L2:ADV7180:enable adv7180 in Android

Move ADV7180 out of choice to make it be enabled with other
camera config simultaneously

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 drivers/media/video/mxc/capture/Kconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/media/video/mxc/capture/Kconfig b/drivers/media/video/mxc/capture/Kconfig
index 243b888..3013155 100644
--- a/drivers/media/video/mxc/capture/Kconfig
+++ b/drivers/media/video/mxc/capture/Kconfig
@@ -99,13 +99,14 @@ config MXC_CAMERA_OV5642
 	---help---
 	  If you plan to use the ov5642 Camera with your MXC system, say Y here.
 
+endchoice
+
 config MXC_TVIN_ADV7180
 	tristate "Analog Device adv7180 TV Decoder Input support"
 	depends on (MACH_MX35_3DS || MACH_MX51_3DS || MACH_MX6Q_SABREAUTO || MACH_MX6Q_ARM2)
 	---help---
 	  If you plan to use the adv7180 video decoder with your MXC system, say Y here.
 
-endchoice
 
 config MXC_CAMERA_OV5640_MIPI
 	tristate "OmniVision ov5640 camera support using mipi"
-- 
1.7.10.4

